browser/main/lib/ThemeManager.js

```diff
@@ -10,8 +10,8 @@ const chooseTheme = ui => {
   const minutes = now.getHours() * 60 + now.getMinutes()
 
   const isEndAfterStart = end > start
-  const isBetweenStartAndEnd = minutes >= start && minutes <= end
-  const isBetweenEndAndStart = minutes >= start || minutes <= end
+  const isBetweenStartAndEnd = minutes >= start && minutes < end
+  const isBetweenEndAndStart = minutes >= start || minutes < end
```

Comment:
I've fixed the problem. It was just the way I was comparing the dates. If you scheduled the change to happen at 15:30, it would change only at 15:31.